meshapp/src/ProfileCardCarousel/Swiper.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useCallback, useEffect } from "react";
+import { useState } from "react";
+import { useSwipeable } from "react-swipeable";
+import { Grid, IconButton } from "@mui/material";
+import profileCard from "./ProfileCard";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+
+function Swiper() {
```

Comment:
This is a pretty large function. Can `Swiper` be converted to a class with smaller methods instead?

Reply:
I've separated several functionalities of Swiper into different functions